docs/howto/deploying-contracts.md

```diff
@@ -0,0 +1,135 @@
+# How to deploy contracts
+
+## Requirements
+
+If you haven't done it, follow the [getting started](../intro/index.md) tutorial first.
+
+## Prepare your environment
+
+Always on master
+```shell
+git checkout master
+```
+
+Fix format of contracts (optional)
+```shell
+yarn format:fix
+```
+
+Update dependencies
+```shell
+yarn
+```
+
+## Deploy contracts
+
+### Fork deployment
+
+The [fork deployment](https://hardhat.org/hardhat-network/guides/mainnet-forking.html) command simulates the state of the selected network and deploy as if we were on this network but locally. Always run the fork deployment first to check what is about to be really deployed and how.
+
+The first argument is the network you want to simulate. 
+The option `--deploy-scripts` overwrites the folders configured in `hardhat.config.ts`.
+In case of fork deployment, the network is actually `hardhat`, so if you want to deploy only on L2, you would have to pass `deploy_polygon` in option to `--deploy-scripts`.
+The option `--tags` selects the deployment script to execute. For instance, `PolygonSand` will only run the deployments with at least the tag `PolygonSand` defined with `func.tags`.
+
+```ts
+func.tags = ['PolygonSand', 'PolygonSand_deploy'];
+```
+
+!!! example Fork deployment of PolygonSand on mumbai
+    ```shell
+    yarn fork:deploy mumbai --deploy-scripts deploy_polygon --tags PolygonSand
+    ```
+
+### Deployment
+
+Once the fork deployment is looking good, you can switch to the real network by running the same command without `fork`. 
+The option `--deploy-scripts` is not needed for a real deployment because the deploy folders are configured correctly in `hardhat.config.ts`
+
+!!! example Deployment of PolygonSand on mumbai
+    ```shell
+    yarn deploy mumbai --tags PolygonSand
+    ```
+
+Once your contract deployed on the network, a deployment file will be generated in the folder
+
+```shell
+./deployments/yournetwork/Contract.json
```

Comment:
maybe try to make it clear that 'network' here means which network we're deploying